app/controllers/events_controller.rb

```diff
@@ -0,0 +1,45 @@
+class EventsController < ApplicationController
+  
+  before_action :logged_in_user, only: :create
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.events.create(creator_id: current_user.id,name: event_params[:name],date: event_params[:date])
```

Comment:
You can omit to assign the `creator_id`. Since you are calling the method from a user instance that field should populate automatically